chains/v2/chains_dev.json

```diff
@@ -2510,5 +2510,41 @@
         "color": "linear-gradient(315deg, #007568 31.86%, #91E3DA 100%)",
         "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Integritee.svg",
         "addressPrefix": 13
+    },
+    {
+        "chainId": "97da7ede98d7bad4e36b4d734b6055425a3be036da2a332ea5a7037656427a21",
+        "name": "Nodle Parachain",
+        "assets": [
+            {
+                "assetId": 0,
+                "symbol": "NODL",
+                "precision": 11,
+                "priceId": "nodle",
```

Comment:
It's seems it's not listed yet, so I would remove that one as well

Reply:
done